### PR TITLE
Updating behaviour docs with correct location for Nuget configfile (windows)

### PR DESCRIPTION
**Pull request by ianrathbone**
As per discussion in #2824 - NuGetDefaults.Config location incorrect, and now updated to location referenced

### CODE REVIEW

File: docs/consume-packages/configuring-nuget-behavior.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ NuGet's behavior is driven by the accumulated settings in one or more `NuGet.Con
...
 | User | **Windows:** `%appdata%\NuGet\NuGet.Config`<br/>**Mac/Linux:** `~/.config/NuGet/NuGet.Config` or `~/.nuget/NuGet/NuGet.Config` (varies by tooling) <br/>Additional configs are supported on all platforms. These configs cannot be edited by the tooling. </br> **Windows:** `%appdata%\NuGet\config\*.Config` <br/>**Mac/Linux:** `~/.config/NuGet/config/*.config` or `~/.nuget/config/*.config` | Settings apply to all operations, but are overridden by any project-level settings. |
-| Computer | **Windows:** `%ProgramFiles(x86)%\NuGet\Config`<br/>**Mac/Linux:** `$XDG_DATA_HOME`. If `$XDG_DATA_HOME` is null or empty, `~/.local/share` or `/usr/local/share` will be used (varies by OS distribution)  | Settings apply to all operations on the computer, but are overridden by any user- or project-level settings. |
+| Computer | **Windows:** `%ProgramFiles(x86)%\NuGet`<br/>**Mac/Linux:** `$XDG_DATA_HOME`. If `$XDG_DATA_HOME` is null or empty, `~/.local/share` or `/usr/local/share` will be used (varies by OS distribution)  | Settings apply to all operations on the computer, but are overridden by any user- or project-level settings. |
```
Comment (nkolev92) [inline docs/consume-packages/configuring-nuget-behavior.md:20]: We should revert this. This is correct.

File: docs/consume-packages/configuring-nuget-behavior.md [hunk 2]
```diff
@@ -236,7 +236,7 @@ The following table describes where the `NuGetDefaults.Config` file should be st
 
 | OS Platform  | `NuGetDefaults.Config` Location |
 | --- | --- |
-| Windows      | **Visual Studio 2017 or NuGet 4.x+:** `%ProgramFiles(x86)%\NuGet\Config` <br />**Visual Studio 2015 and earlier or NuGet 3.x and earlier:** `%PROGRAMDATA%\NuGet` |
+| Windows      | **Visual Studio 2017 or NuGet 4.x+:** `%ProgramFiles(x86)%\NuGet` <br />**Visual Studio 2015 and earlier or NuGet 3.x and earlier:** `%PROGRAMDATA%\NuGet` |
```
Comment (nkolev92) [inline docs/consume-packages/configuring-nuget-behavior.md:239]: This is a good change.

**Review by nkolev92 — CHANGES_REQUESTED** [review]
I think we're changing too many things.

The directory for machine wide is correct. The one to which we're changing it right now is the NuGetDefaults.config location.

File: docs/consume-packages/configuring-nuget-behavior.md (hunk 2 above)
Comment (ianrathbone) [reply docs/consume-packages/configuring-nuget-behavior.md:239]: Thank you I have left this in

File: docs/consume-packages/configuring-nuget-behavior.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ nuget config -set repositoryPath=c:\packages -configfile c:\my.Config
 nuget config -set repositoryPath=c:\packages -configfile .\myApp\NuGet.Config
 
 # Set repositoryPath in the computer-level file (requires elevation)
-nuget config -set repositoryPath=c:\packages -configfile %ProgramFiles(x86)%\NuGet\Config\NuGet.Config
+nuget config -set repositoryPath=c:\packages -configfile %ProgramFiles(x86)%\NuGet\NuGet.Config
```
[the remark this answers is not in the document]
Comment (nkolev92) [reply docs/consume-packages/configuring-nuget-behavior.md:69]: The expected configs are:

- %ProgramFiles(x86)%\NuGet\Config\NuGet.Config - Machine Wide
- %ProgramFiles(x86)%\NuGet\NuGetDefaults.Config - Defaults config. 

This change would currently point to a config that wouldn't normally exist or be used.

File: docs/consume-packages/configuring-nuget-behavior.md [hunk 4]
```diff
@@ -38,7 +38,7 @@ If the tooling you use involves both locations, consider consolidating them by f
 
 Notes for earlier versions of NuGet:
 - NuGet 3.3 and earlier used a `.nuget` folder for solution-wide settings. This folder is not used in NuGet 3.4+.
-- For NuGet 2.6 to 3.x, the computer-level config file on Windows was located in `%ProgramData%\NuGet\Config[\{IDE}[\{Version}[\{SKU}]]]\NuGet.Config`, where `{IDE}` can be `VisualStudio`, `{Version}` was the Visual Studio version such as `14.0`, and `{SKU}` is either `Community`, `Pro`, or `Enterprise`. To migrate settings to NuGet 4.0+, simply copy the config file to `%ProgramFiles(x86)%\NuGet\Config`. On Linux, this previous location was `/etc/opt`, and on Mac, `/Library/Application Support`.
+- For NuGet 2.6 to 3.x, the computer-level config file on Windows was located in `%ProgramData%\NuGet\Config[\{IDE}[\{Version}[\{SKU}]]]\NuGet.Config`, where `{IDE}` can be `VisualStudio`, `{Version}` was the Visual Studio version such as `14.0`, and `{SKU}` is either `Community`, `Pro`, or `Enterprise`. To migrate settings to NuGet 4.0+, simply copy the config file to `%ProgramFiles(x86)%\NuGet`. On Linux, this previous location was `/etc/opt`, and on Mac, `/Library/Application Support`.
```
Comment (nkolev92) [inline docs/consume-packages/configuring-nuget-behavior.md:41]: Do we need this change?

Comment (nkolev92) [inline docs/consume-packages/configuring-nuget-behavior.md:41]: I think this one is not correct is it?

The machine wide config is correct, it's the defaults config that isn't.

**Review by nkolev92 — COMMENTED** [review]
Thanks for responding.

Apologies, I found something new.

File: docs/consume-packages/configuring-nuget-behavior.md (hunk 4 above)
Comment (ianrathbone) [reply docs/consume-packages/configuring-nuget-behavior.md]: No problem at all, I have reverted both of these changes so only the single change is in place 😊

**Comment by ianrathbone** [comment]
> Thank you for the contribution @ianrathbone and your patience with our reviews :D

My pleasure - I hope to do some more!

**Review by erdembayar — APPROVED** [review]

**Pull request merged**